EIPS/eip-template.md

```diff
@@ -0,0 +1,59 @@
+---
```

Comment:
```suggestion
---
eip: 7733
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.